### ISSUE TITLE
New testclient, websocket connection handling

### ISSUE THREAD

**Issue by SebastianSchildt** [eclipse/kuksa.val#22]
Something seems to be wrong with websocket connection handling

Reproduce

- Start server
- Connect with new testclient (connect authorize setValue)
- works fine
- Restart server (so the old connection is broken/invalid)
- in testclient type "connect"
- authorize: That works! (so I assume it DID reconnect?)
 - setValue: first time works
 - setValue second time: Exception: websockets.exceptions.ConnectionClosedError: code = 1006 (connection closed abnormally [internal]), no reason
 - Ctrl-C gives back prompt
 - setValue works again

Already not perfect, but now the super strange thing:
type `connect` again: Looks fine, but now roughly every second call to setValue succeeds while the others give 403
Do we have two sessions now, that are randomly used

@nayakned Can you check, and maybe push a fix?

**Comment by wenwenchenbosch** [comment]
A new robust problem found:

 - Start server
 - Start testclient
 - In testclient type `Connect` two times (second time maybe by mistake)
 - Authorize
 
works fine until now, but then setValue may be failed with the following error message:

```
VSS Client> setValue Vehicle.Speed 500
{
    "action": "set",
    "error": {
        "message": "Path(s) in set request do not have write access or is invalid",
        "number": 403,
        "reason": "Forbidden"
    },
    "requestId": "1235",
    "timestamp": 1596814834
}
```

**Comment by nayakned** [comment]
Try Vehicle.OBD.Speed if the problem persists.

**Comment by SebastianSchildt** [comment]
Closed by #28
